vpr/src/draw/intra_logic_block.cpp

```diff
@@ -578,14 +598,34 @@ void draw_logical_connections(ezgl::renderer* g) {
             const t_pb_graph_node* sink_pb_gnode = atom_ctx.lookup.atom_pb_graph_node(sink_blk_id);
             ClusterBlockId sink_clb = atom_ctx.lookup.atom_clb(sink_blk_id);
 
+            int sink_layer_num = place_ctx.block_locs[sink_clb].loc.layer;
```

Comment:
Suggest factoring this into a helper routing that returns a bool of whether or not to draw and a transparency (probably best to return a t_draw_layer_display).